drake/systems/test/testDrakeSystemFrames.cpp

```diff
@@ -0,0 +1,39 @@
+
+#include <iostream>
+#include "DrakeSystem.h"
+
+using namespace std;
+
+int main(int argc, char* argv[]) {
+
+  shared_ptr<DrakeSystem> test_system1( new DrakeSystem("MySystem",
```

Comment:
`make_shared` is shorter and preferred because it handles exceptions in the constructor better.